### DISCUSSION TITLE
Is it possible to change sync frequency of all feeds with  a particular tag using command line API

### DISCUSSION THREAD

**Discussion by lamyergeier** [FreshRSS/FreshRSS#6260] [Ideas]
Is it possible to change sync frequency of all feeds with  a particular tag using command line API? I would like to change all feeds with tag 'youtube' to sync once in 72 hours, while others can sync normally.

Is there a Command line API for this? Or may be possible to do it with we interface (GUI) without having to manually change for one feed at a time?

**Comment by Alkarex** [comment]
Not at the moment, but PR welcome. At the moment, it would require an SQL command.
